src/app/component/user/components/profile/users-friends/friend-dashboard/all-friends/all-friends.component.ts

```diff
@@ -0,0 +1,127 @@
+import { Component, OnDestroy, OnInit } from '@angular/core';
+import { MatSnackBarComponent } from '@global-errors/mat-snack-bar/mat-snack-bar.component';
+import { LocalStorageService } from '@global-service/localstorage/local-storage.service';
+import { FriendArrayModel, FriendModel, } from '@global-user/models/friend.model';
+import { UserFriendsService } from '@global-user/services/user-friends.service';
+import { Subject } from 'rxjs';
+import { catchError, takeUntil } from 'rxjs/operators';
+
+@Component({
+  selector: 'app-all-friends',
+  templateUrl: './all-friends.component.html',
+  styleUrls: ['./all-friends.component.scss']
+})
+export class AllFriendsComponent implements OnInit, OnDestroy {
+
+  public userId: number;
+  public Friends: FriendModel[];
+  private destroy = new Subject();
```

Comment:
As I understand, this construction should be:
private destroy$: ReplaySubject<boolean> = new ReplaySubject(1);

Reply:
On my view construction new Subject() is posible too.